pkg/dao/types/status/status_servicerevision.go

```diff
@@ -0,0 +1,40 @@
+package status
+
+const (
+	ServiceRevisionStatusReady ConditionType = "Ready"
```

Comment:
Only ready status here is too weak to represent different states of revision. Ready used to represent final status.

Reply:
Currently, service revision only contain 3 status, running, succeed and failed, correspond to the 3 types of the ServiceRevisionStatusReady condition one by one. May be add more conditions or status in the future, but not in this pull request.